apps/sms/views/conversation/js/conversation.js

```diff
@@ -763,15 +767,14 @@ var ConversationView = {
           }
 
           this.recipients.add(recipient);
-        }.bind(this));
-      }, this);
+
+          this.draft.isEdited = false;
```

Comment:
Absence of this line were causing draft with recipients to be always marked as "isEdited=true".